ci/jenkins/test.sh

```diff
@@ -658,7 +663,7 @@ function deliver_antrea {
         kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 !~ role {print $6}' | while read IP; do
             rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" antrea-ubuntu.tar jenkins@[${IP}]:${WORKDIR}/antrea-ubuntu.tar
             rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" flow-aggregator.tar jenkins@[${IP}]:${WORKDIR}/flow-aggregator.tar
-            ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; ${PRINT_DOCKER_STATUS}; docker load -i ${WORKDIR}/antrea-ubuntu.tar; docker load -i ${WORKDIR}/flow-aggregator.tar" || true
+            ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; ${PRINT_DOCKER_STATUS}; docker load -i ${WORKDIR}/antrea-ubuntu.tar; docker load -i ${WORKDIR}/flow-aggregator.tar; ctr -n=k8s.io images import ${DEFAULT_WORKDIR}/antrea-ubuntu.tar; ctr -n=k8s.io images import ${DEFAULT_WORKDIR}/flow-aggregator.tar;ctr -n k8s.io images import ${WORKDIR}/antrea-ubuntu.tar;ctr -n k8s.io images import ${WORKDIR}/flow-aggregator.tar" || true
```

Comment:
Why needs to load the images to both docker and containerd?

Reply:
I was thinking we don't need to assume worker nodes' container runtime. Load the images both won't hurt. The command will print error message such as "docker ...: command not found" if use containerd and continue the compile process. Please check the update commit from this pr https://github.com/antrea-io/antrea/pull/5069